cmd/evm/internal/t8ntool/transition.go

```diff
@@ -564,7 +564,7 @@ func VerkleKeys(ctx *cli.Context) error {
 			CodeHash: crypto.Keccak256Hash(acc.Code).Bytes(),
 			Root:     common.Hash{},
 		}
-		err := vkt.UpdateAccount(addr, account)
```

Comment:
tbh I don't think this method of passing the code size here works, but it's a problem with the base branch that I need to look into, not a showstopper for this PR.